src/TestSuite.cls

```diff
@@ -33,21 +33,24 @@ End Enum
 Public Event BeforeEach(Test As TestCase)
 Public Event Result(Test As TestCase)
 Public Event AfterEach(Test As TestCase)
+Public Event Group(Suite As TestSuite)
 
 ''
 ' (Optional) description of suite for display in runners
 '
 ' @property Description
 ' @type String
 ''
-Public Description As String
+Public Name As String
```

Comment:
what do you think about keeping `Description` as a property, for backwards compatibility.

```vb
Public Property Get Description() as String
     Description = Name
End Property
Public Property Let Description(NewDescription as String)
     Name = NewDescription
End Property
```

this would reduce the need to alter existing code for a semantic name change.

Reply:
:+1: